src/react/deckgl.js

```diff
@@ -38,6 +38,7 @@ const propTypes = {
   effects: PropTypes.arrayOf(PropTypes.instanceOf(Effect)),
   gl: PropTypes.object,
   debug: PropTypes.bool,
+  pickingRadius: PropTypes.number,
```

Comment:
Out of curiosity, what use cases do you imagine for exposing this? Couldn't hurt, I suppose, just thinking out loud